app/src/features/rules/screens/rulesList/components/RulesList/components/RulesColumn/RulesColumn.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import { isRule } from "features/rules/utils";
+import { RuleTableRecord } from "../RulesTable/types";
+import { Link } from "react-router-dom";
+import { Button, Progress, Tooltip } from "antd";
+import { RiInformationLine } from "@react-icons/all-files/ri/RiInformationLine";
+import { RuleSelectionListDrawer } from "../RuleSelectionListDrawer/RuleSelectionListDrawer";
+import PATHS from "config/constants/sub/paths";
+import { RecordStatus } from "features/rules/types/rules";
+import { useTheme } from "styled-components";
+import { SOURCE } from "modules/analytics/events/common/constants";
+import { trackNewRuleButtonClicked } from "modules/analytics/events/common/rules";
+import { useDispatch } from "react-redux";
+import { actions } from "store";
+
+const RulesColumn: React.FC<{
```

Comment:
Rename this to `RuleNameColumn`

Reply:
Also keep this component inside `features/rules/screens/rulesList/components/RulesList/components/RulesTable/components`